### DISCUSSION TITLE
`ssl_check_hostname` option for MySQLdb dialect

### DISCUSSION THREAD

**Discussion by zhangyangyu** [sqlalchemy/sqlalchemy#9084] [Usage Questions] [answered]
On the doc https://docs.sqlalchemy.org/en/20/dialects/mysql.html#ssl-connections, it says MySQLdb dialect supports `ssl_check_hostname`, but actually the mysqlclient driver doesn't support a `check_hostname` option, like PyMySQL. Actually mysqlclient uses `ssl_mode` to set verify behaviour. So I think this `ssl_check_hostname` won't work. Am I missing something?

**Comment by zzzeek** [comment, marked as answer]
> Am I missing something?

doesn't seem that way.   this note should be moved to the pymysql section.

**Reply by CaselIT** [reply]
I've edited in https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4353

**Comment by zhangyangyu** [comment]
@zzzeek @CaselIT This seems being deliberately changed in https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2774. You might want to take a look.

**Reply by zzzeek** [reply]
it's fine that it's still accepted in the base "mysqldb" dialect, not totally ideal but wouldn't cause new problems, it gets passed to mysqlclient and rejected (or ignored) which is not any different from what would happen anyway.
